prac2-pages/chhotu/lib/main.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MaterialApp(
+      home: Project(),
+    ));
+
+class Project extends StatefulWidget {
+  const Project({super.key});
+
+  @override
+  State<Project> createState() => _ProjectState();
+}
+
+class _ProjectState extends State<Project> {
+  bool value = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: <Widget>[
+          Container(
+            margin: EdgeInsets.only(top: 200.0, left: 135.0),
+            child: Column(
+              children: <Widget>[
+                Text(
+                  'Login',
+                  style: TextStyle(
+                      color: Colors.black, fontSize: 20.0, height: 1.0),
+                ),
+                Text(
+                  'Create new account',
+                  style: TextStyle(height: 3, color: Colors.blue),
+                ),
+              ],
+            ),
+          ),
+          Container(
+            padding: EdgeInsets.only(
+                top: MediaQuery.of(context).size.height * 0.4,
+                right: 20.0,
+                left: 20.0),
+            child: Column(
+              children: [
+                Padding(
+                  padding: EdgeInsets.only(right: 270.0),
```

Comment:
No need to use padding to align the item to the left of the screen. :exclamation: 

You can use Column widget and set the **mainAxisAlignment** property to start. And the issue would be resolved!